source/gen_security.rst

```diff
@@ -316,7 +316,7 @@ use more general `Access based access control`_. For this rewrite `core.rego` fi
 
 
 .. code-block:: javascript
-   :name: ODAHU Role base access implementation
+   :name: ODAHU Role based access implementation
```

Comment:
```suggestion
   :name: ODAHU Role-based access implementation
```

Reply:
@BPylypenko I would also write it with hyphen, but it is spelled without it everywhere else in the doc. Do we want to replace it everywhere to keep it consistent? :)9